controllers/mlmd.go

```diff
@@ -57,7 +60,24 @@ func (r *DSPAReconciler) ReconcileMLMD(dsp *dspav1alpha1.DataSciencePipelinesApp
 			return err
 		}
 	} else {
-		err := r.ApplyDir(dsp, params, mlmdTemplatesDir)
+		err := r.ApplyDir(dsp, params, mlmdTemplatesDir+"/"+mlmdGrpcService)
```

Comment:
Why did we need to move the two service templates to their own subdirectory? Is it so that we can trick OpenShift into giving us our certs first? (The services are what have that annotation to make it hand over the certs.)

Might be worth a comment.

Reply:
Done.